src/domain/authentication/index.types.d.ts

```diff
@@ -39,3 +40,9 @@ interface IAuthWithPhonePasswordlessService {
     password: IdentityPassword
   }): Promise<IdentityPhone | AuthenticationError> // TODO: should be IdentityPhoneWithPassword
 }
+
+interface IIdentityRepository {
+  getIdentity(id: KratosUserId): Promise<IdentityPhone | KratosError>
```

Comment:
A bit inconsistent naming here. If we are using `IdentityRepository` as then it should be `IdentityId` and `IdentityError` - not `Kratos`...

Reply:
yeah I've been switching back and forth on the fact it's should be generic (`Identity`) or dedicated to Kratos. unclear to me how we should handle this, but I've started with Kratos and moved to Identity